src/utils/hints/accountHints.js

```diff
@@ -0,0 +1,14 @@
+export const accountHints = {
+  balance: 'Amount of AE owned by the account.',
+  value: 'Fiat value of AE owned by the account.',
+  transactions: 'Amount of transactions where the account is involved.',
+  aensNames: 'Amount of Names owned by the account.',
+  nonce: 'Value of the nonce for the account. Nonce serves as a one-time authenticator for transactions.',
+  apiLinks: 'Node API link of the account.',
+  hash: 'Transaction hash of transaction where the account is involved.',
```

Comment:
`Transaction hash of the transaction where the account was involved.` sounds better to me.